src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt

```diff
@@ -369,8 +369,9 @@ class ModuleRegistry(jiraClient: JiraClient, private val config: Config) {
             UpdateLinkedModule()
         ) { issue ->
             UpdateLinkedModule.Request(
+                issue.reporter.name,
                 issue.issueLinks
-                    .map { it.type.name },
+                    .map { UpdateLinkedModule.IssueLink(it.inwardIssue.reporter.name, it.type.name) },
```

Comment:
This will not work.
inwardIssue has only a very limited amount of fields. This is the api response I got from a ticket:
```json
"issuelinks": [
	{
		"id": "161296",
		"self": "https://bugs.mojang.com/rest/api/2/issueLink/161296",
		"type": {
			"id": "10102",
			"name": "Duplicate",
			"inward": "is duplicated by",
			"outward": "duplicates",
			"self": "https://bugs.mojang.com/rest/api/2/issueLinkType/10102"
		},
		"inwardIssue": {
			"id": "241711",
			"key": "BDS-884",
			"self": "https://bugs.mojang.com/rest/api/2/issue/241711",
			"fields": {
				"summary": "Minecarts despawning",
				"status": {
					"self": "https://bugs.mojang.com/rest/api/2/status/5",
					"description": "A resolution has been taken, and it is awaiting verification by reporter. From here issues are either reopened, or are closed.",
					"iconUrl": "https://bugs.mojang.com/images/icons/statuses/resolved.png",
					"name": "Resolved",
					"id": "5",
					"statusCategory": {
						"self": "https://bugs.mojang.com/rest/api/2/statuscategory/3",
						"id": 3,
						"key": "done",
						"colorName": "green",
						"name": "Done"
					}
				},
				"issuetype": {
					"self": "https://bugs.mojang.com/rest/api/2/issuetype/1",
					"id": "1",
					"description": "A problem which impairs or prevents the functions of the product.",
					"iconUrl": "https://bugs.mojang.com/secure/viewavatar?size=xsmall&avatarId=18903&avatarType=issuetype",
					"name": "Bug",
					"subtask": false,
					"avatarId": 18903
				}
			}
		}
	}
]
```

I'm not sure what the best solution to this is. Getting every linked issue on every ticket would generate a lot of traffic and is probably not worth the afford.
I think it would be better for #138 to receive a function that gets (and maps) an issue to a module data class and assert that one of the duplicates is not from the one with the same reporter.

Reply:
Oops, that's too sad. I'd prefer just raise the threshold of the Linked field in #138🤦